### DISCUSSION TITLE
Can`t run or upload the Code

### DISCUSSION THREAD

**Discussion by Meo98** [pycom/pymakr-vsc#247] [Q&A]
Hello, I use a Firebeetle Esp32 and Pymakr on VsCode. Pymakr recognizes the device and I can also print ("Hello World") in the terminal and the Esp32 prints it out. However, as soon as I write a program and upload it, the program says that the COM port is busy. After that, the program no longer recognizes the port and I have to reload the firmware of the Esp32. I didn't have these problems with the old Pymakr software. The code I loaded was:

from machine import Pin, PWM
pwm = PWM(Pin(27), freq=2000, duty=512)

**Comment by jakobrosenberg** [comment]
Please see this https://github.com/pycom/pymakr-vsc/issues/242#issuecomment-1140751462
